article/app/controllers/ArticleController.scala

```diff
@@ -121,6 +122,10 @@ class ArticleController(contentApiClient: ContentApiClient)(implicit context: Ap
         else if (article.article.isExplore) views.html.articleExplore(article)
         else if (article.article.isImmersive) views.html.articleImmersive(article)
         else if (request.isAmp) views.html.articleAMP(article)
+        else if (article.article.isRecipeArticle && false) {
```

Comment:
Instead of staying false, maybe as part of the `isRecipeArticle` you can have the a switch or test which by default is false.